### DISCUSSION TITLE
Show top #3 response emojis to a thread to help prioritization

### DISCUSSION THREAD

**Discussion by tomkerkhove** [community/community#2889] [Discussions]
Show top #3 response emojis to a thread to help prioritization of topics/concerns/requests in projects.

For example, today it looks like this:
![image](https://user-images.githubusercontent.com/4345663/101597794-7a825800-39f7-11eb-8d06-06e5be3ca156.png)

While when going in detail, you'll see that certain people support the idea which is not indicated above:
![image](https://user-images.githubusercontent.com/4345663/101597925-a7cf0600-39f7-11eb-8597-8605ffaa8a8b.png)

**Comment by becca** [comment]
Do you use our web notifications, by chance? Curios if this index view `/discussions` is one of your main "triage" views or whether notifications is where you first triage a lot of Discussions related activity.
